app/styles/app.scss

```diff
@@ -2,12 +2,17 @@
 @import 'variables';
 @import 'bootstrap';
 @import 'ember-power-select';
+@import 'daterangepicker.scss';
+@import 'c3';
```

Comment:
Some import is causing a deprecation warning:
```
DEPRECATION WARNING on line 7, column 8 of /code/tmp/sass_compiler-input_base_path-0ZTDxWsh.tmp/0/app/styles/app.scss:
Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
Use a custom importer to maintain this behaviour. Check your implementations documentation on how to create a custom importer.
```

I think it might be [hint.css](https://github.com/CenterForOpenScience/ember-osf-preprints/pull/585/files#diff-2fe2a7faee7de1e0d5d82e032498d84cR66)

Reply:
This can be taken care of as a separate ticket as it wasn't caused by my refactor.

Reply:
I feel like I've seen that one for a while. I have vague recollections of trying to solve but having issues. Either way is fine by me.